packages/backend/constants/common.ts

```diff
@@ -34,6 +34,7 @@ export const DEFAULT_SCOPE = {
     [TP_ID.zohocrm]: ['ZohoCRM.modules.ALL', 'ZohoCRM.settings.ALL', 'ZohoCRM.users.ALL', 'AaaServer.profile.READ'],
     [TP_ID.sfdc]: [], // https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_tokens_scopes.htm&type=5
     [TP_ID.pipedrive]: [],
+    [TP_ID.slack]: ['users:read', 'users.profile:read'],
```

Comment:
Do we not need the `channels:read` or `channels:write` permission here?

Reply:
At least for retrieving channels it worked without those permissions.